kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiReservedKeys.scala

```diff
@@ -24,6 +24,7 @@ object KyuubiReservedKeys {
   final val KYUUBI_SESSION_SIGN_PUBLICKEY = "kyuubi.session.sign.publickey"
   final val KYUUBI_SESSION_USER_SIGN = "kyuubi.session.user.sign"
   final val KYUUBI_SESSION_REAL_USER_KEY = "kyuubi.session.real.user"
+  final val KYUUBI_SESSION_BATCH_RESOURCE_UPLOADED_KEY = "kyuubi.session.batch.resource.uploaded"
```

Comment:
Should it be `kyuubi.session.batch` instead of `kyuubi.batch.session`? As we already have `kyuubi.batch.session.idle.timeout`, they should be consistent.

cc @bowenliang123 @turboFei

Reply:
Or just remove "session", and use the "kyuubi.batch." namespace.

Reply:
SGTM @bowenliang123 @pan3793